### DISCUSSION TITLE
Clearly guide on install and configure Podcastfy?

### DISCUSSION THREAD

**Discussion by toilv** [souzatharsis/podcastfy#188] [General]
I have already installed Podcastfy and requirements but take time. I am now not able to know which part of files edit configurations.
Could you update guide for more detail?

**Comment by souzatharsis** [comment]
Hi,

Please take a look at the accompanying Python notebook:

podcastfy.ipynb

https://github.com/souzatharsis/podcastfy/blob/df718eb1ec80d7b8348d06d8038eb2c333048d78/podcastfy.ipynb
